src/Form/Dropdown/FormActorsDropdown.php

```diff
@@ -72,7 +72,88 @@ protected static function getTypes(): array
     public static function show(string $name, array $values, array $params = []): string
     {
         $params['width'] = '100%';
+        $params['templateSelection'] = <<<JS
+            function (data) {
+                let icon = '';
+                let text = data.text;
+                let title = data.title;
+                if (data.id != undefined && data.id.startsWith('users_id-') || data.text === 'User') {
+                    icon = '<i class="ti fa-fw ti-user mx-1" title="User"></i>';
+                } else if (data.id != undefined && data.id.startsWith('groups_id-') || data.text === 'Group') {
+                    icon = '<i class="ti fa-fw ti-users mx-1" title="Group"></i>';
+                } else if (data.id != undefined && data.id.startsWith('suppliers_id-') || data.text === 'Supplier') {
+                    icon = '<i class="ti fa-fw ti-package mx-1" title="Supplier"></i>';
+                }
```

Comment:
I guess that tests like `data.text === 'User'` will work only on english languages. The options texts will be different in other languages.

You should update both the way to check this and the displayed icon title.